pbconf/python-redfish.pb

```diff
@@ -130,7 +130,7 @@ defpkgdir python-redfish = .
 
 # List of files per pkg on which to apply filters
 # Files are mentioned relatively to pbroot/defpkgdir
-filteredfiles python-redfish = redfish-client/redfish-client,doc/source/conf.py
+filteredfiles python-redfish = redfish-client/redfish-client,doc/source/conf.py,setup.py,./redfish-client/etc/redfish-client.conf,install.sh
```

Comment:
This is for pip installation. I will have to discuss that.
I agree that this is not the ideal approach but currently I had to do that.
That could be improved if we will update pb to produce Pypi "packages".